### PR TITLE
Make db connection pool configurable

### PR DESCRIPTION
**Pull request by tej99**
To improve performance of the system the db connection pool needs to be increased to sustain the load. This PR allows us to configure the pool size with environment variables.

### CODE REVIEW

**Comment by tej99** [comment]
[Optimise DB Connection Settings](https://trello.com/c/SwXkyTRo/44-optimise-db-connection-settings)

File: test/test_create_database.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ def test_postgres_database_schema_exists(self):
         db_schema = 'partysvc'
 
         with patch('run.create_engine'), patch('run.scoped_session'), patch('run.command.upgrade') as upgrade:
-            create_database(db_connection, db_schema)
+            create_database(db_connection, db_schema, 5, 10, -1)
```
Comment (gemmai95) [inline test/test_create_database.py:19]: Are these numbers meant to be configurable by the env vars?

Comment (tej99) [reply test/test_create_database.py:19]: Keeping them hard coded here is fine as the tests focus on the alembic stuff.

File: config.py [hunk 2]
```diff
@@ -121,6 +125,9 @@ class TestingConfig(DevelopmentConfig):
     REQUESTS_GET_TIMEOUT = 99
     REQUESTS_POST_TIMEOUT = 99
     DATABASE_SCHEMA = 'partysvc'
+    DB_POOL_SIZE = 5
+    DB_MAX_OVERFLOW = 10
+    DB_POOL_RECYCLE = -1
```
Comment (andrewmil) [inline config.py]: We don't need these in the Testing config as they'll use the default values in the Config class

**Review by andrewmil — APPROVED** [review]
:+1: Looks good

**Review by gemmai95 — APPROVED** [review]

**Pull request merged**